lib/gui/pages/main/controllers/flash.js

```diff
@@ -111,9 +111,12 @@ module.exports = function(
       return 'Flash!';
     }
 
-    if (flashState.percentage === 0 && !flashState.speed) {
+    const PERCENTAGE_MINIMUM = 0;
+    const PERCENTAGE_MAXIMUM = 100;
+
+    if (flashState.percentage === PERCENTAGE_MINIMUM && !flashState.speed) {
       return 'Starting...';
-    } else if (flashState.percentage === 100) {
+    } else if (flashState.percentage === PERCENTAGE_MAXIMUM) {
       if (isChecking && SettingsModel.get('unmountOnSuccess')) {
         return 'Unmounting...';
       }
```

Comment:
There's multiple separate `if ...` blocks here. Should they be combined into a single `if ... else if ... else if ... else ...` block?